src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const thousandsSeparator = (n) => n.toLocaleString('en-Us');
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+const pop = document.querySelectorAll('.population');
+const arrPop = [...pop].map(el => el.innerText);
+const arrPopToNum = arrPop.map(el => +el.split(',').join(''));
+const len = arrPopToNum.length;
+const totPop = arrPopToNum.reduce((acc, el) => acc + el);
+const averagePop = totPop / len;
```

Comment:
do not use 1 letter variables and do not shorten var names
use meaningful ones instead